src/cli.rs

```diff
@@ -84,6 +84,29 @@ pub fn build_cli() -> App<'static, 'static> {
                 SubCommand::with_name("list")
                     .visible_alias("ls")
                     .about("List CloudTruth templates"),
+                SubCommand::with_name("getit")
```

Comment:
Unfortunately, there's no lint rule for it (I think we'd need to write a custom script), but I was trying to keep the subcommands sorted alphabetically.